dashboard/test/mailers/ops_mailer_test.rb

```diff
@@ -15,4 +15,114 @@ class OpsMailerTest < ActionMailer::TestCase
     assert_match "added 1 teachers", mail.body.encoded
   end
 
+  test "workshop_in_2_weeks_reminder" do
+    # Create a workshop that starts 14 days from now. Include other segments for authenticity
+    @workshop_1 = create(:workshop, phase: 2, cohorts: [create(:cohort, id: 1,
+      teachers: [create(:teacher, email: 'abc@code.org'), create(:teacher, email: 'xyz@code.org')])],
+      facilitators: [create(:facilitator, email: 'facilitator_1@code.org')])
+    @workshop_1.segments.clear
+    @workshop_1.segments << create(:segment, workshop_id: @workshop_1.id, start: DateTime.now + 14.day, end: DateTime.now + 14.day + 60.minute)
```

Comment:
Rather than starting precisely at midnight, could you change this to later in the day (e.g. 9am) and make sure it is still correctly selected.